tests/core/test_integration.py

```diff
@@ -718,7 +723,7 @@ def test_incremental_time_self_reference(
     )
     # Validate that we made 7 calls to the customer_revenue_lifetime snapshot and 1 call to the customer_revenue_by_day snapshot
```

Comment:
Update this comment from `7` to `6`. Also, if you don't find the comment helpful, you could remove it.